tests/serving-functions-go.test.js

```diff
@@ -102,12 +94,14 @@ require github.com/aws/aws-lambda-go v1.20.0`,
               async () => {
                 const response = await got(`http://localhost:${port}/.netlify/functions/go-func`).text()
 
-                return response === 'Hello, Netlify!'
+                return response === updatedBody
               },
               { interval: WAIT_INTERVAL, timeout: WAIT_TIMEOUT },
             ),
           outputBuffer,
         )
+
+        await removeExecaMock()
```

Comment:
Should this be in a `try`/`finally` block? To ensure cleanup is performed even if an exception is thrown before this.

Reply:
Done in https://github.com/netlify/cli/pull/3047/commits/5426129adb724acfc244e626fea85c5eab2c4661.